corehq/apps/cloudcare/static/cloudcare/js/preview_app/main.js

```diff
@@ -1,22 +1,25 @@
 hqDefine("cloudcare/js/preview_app/main", function () {
+    var initialPageData = hqImport("hqwebapp/js/initial_page_data"),
+        PreviewApp = hqImport("cloudcare/js/preview_app/preview_app"),
+        Utils = hqImport("cloudcare/js/utils");
```

Comment:
I thought our naming convention only used `UpperFirst` for classes, and other variables should use `lowerFirst`. How did you decide to use `UpperFirst` rather than `lowerFirst` for module names? Why is `initialPageData` different?

Reply:
The conventions are messy in cloudcare, which tends to use caps for imported modules. But it felt weird to type `InitialPageData` since we don't spell it that way anywhere else in HQ.

I can take a pass at cleaning up the naming conventions.

Reply:
FWIW, my preference is `lowerFirst` for things that are not classes since I think that aligns most closely with our style guidance.

Reply:
I added a few commits of variable renames. They aren't comprehensive, and cloudcare is confusing because module definitions aren't consistent (some js files are a single class). But this PR should no longer be making the naming situation worse, and I cleaned up some pre-existing issues.